integration/src/main/resources/testusers/scarlett.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "scarlett",
+    "userEmail": "Scarlett.Flowerpicker@test.firecloud.org",
```

Comment:
This account is used for automated testing in the Terra UI, and it already exists in all of the pre-prod environments (which makes setup a lot easier for us). @ichengchang let me know if there is a different account you would prefer we use for any reason.

Reply:
This is totally fine. The choice of test users is not mandated by `TestRunner`, developers are free to choose test users that are the most convenient to set up.